docs/how_to.md

```diff
@@ -0,0 +1,169 @@
+# Documentation on Creating Coinswap
```

Comment:
Maybe we can call this an "Example Coinswap with `teleport` cli app". 

We might not have this binary later. We will make sure to update this example accordingly.